sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsConf.scala

```diff
@@ -928,6 +928,14 @@ object RapidsConf {
     .booleanConf
     .createWithDefault(false)
 
+  val _2GB: Long = 2L * 1024 * 1024 * 1024
+  val APPROX_PAR_META_DATA: Int = 10 * 1024 * 1024 // we are estimating 10MB
```

Comment:
Rather than pollute the public members of this class with essentially temporary values, I'd rather see something like this:
```
private val TEST_CACHE_BATCH_SIZE_DEFAULT = {
  val _2gb = 2L * 1024 * 1024 * 1024
  val approx_par_meta_data = 10 * 1024 * 1024
  _2gb - approx_par_meta_data
}